### PR TITLE
Inherit `style` attribute

### PR DESCRIPTION
**Pull request by distantnative**
Everywhere where we set `inheritAttrs: false`, we need to ensure to manually inherit `class` and `style` attributes then.

### CODE REVIEW

**Comment by distantnative** [comment]
Don't merge while we explore option to remove `inheritAttrs: false` completely

**Comment by distantnative** [comment]
We will keep `inheritAttrs: false`, so this can be reviewed again.

File: panel/src/components/Forms/Blocks/Block.vue [hunk 1]
```diff
@@ -13,8 +14,8 @@
 		:data-last-selected="isLastSelected"
 		:data-selected="isSelected"
 		:data-translate="fieldset.translate"
-		class="k-block-container"
 		:tabindex="isDisabled ? null : 0"
+		:style="$attrs.style"
```
Comment (bastianallgeier) [inline panel/src/components/Forms/Blocks/Block.vue]:
```suggestion
		:style="$attrs.style"
		:tabindex="isDisabled ? null : 0"
```

**Review by bastianallgeier — CHANGES_REQUESTED** [review]
Just a small change.

**Review by bastianallgeier — APPROVED** [review]

**Comment by bastianallgeier** [comment]
I think we need to move forward after v5 by actively describing, which components should be able to set the style attribute at all. It's a great comporomise like this and I'm happy that we can get rid of inheritAttributes, but it does not make sense either to simply allow it everywhere. I think that's the strength of this approach. We have more control and transparency of what is actually possible.

**Comment by distantnative** [comment]
@bastianallgeier just for clarification, we can't get rid of inheritAttrs

**Comment by bastianallgeier** [comment]
I put it the wrong way. I mean we get rid of inheritAttributes: true and avoid injecting unwanted non-valid attributes.

**Pull request merged**